docs/robots/warthog/user_manual_warthog.mdx

```diff
@@ -4,7 +4,671 @@ sidebar_position: 2
+  Ensure that the amount of overhanging payload allows the UGV to operate safely and does not contact the terrain, especially when crossing steep and/or deep gullies.
+- The available internal chassis volume is:
+  - Length of 445 mm (17.5")
+  - Width of 660 mm (26")
+  - Height of 241 mm (9.5").
+    This space is located at the center of the chassis between two battery packs.
+    Consider that anything placed inside the chassis mush be secured as to not move or shift during UGV operation.
+    Any payload secured ins
+
+:::note
+
+Permanent damage resulting from custom modifications to the mounting plate is not covered under warranty and may not be supported by Clearpath Support.
+Please contact our Support team if you require assistance or have any questions relating to custom modifications.
+
+:::
+
+### Electrical Integration {#warthog-electrical-integration}
+
+:::danger
+
+For continued protection against risk of fire, always replace fuses only with those of the same type and rating.
+
+:::
+
+The user power receptacles located in the User Panel are capable of supplying 12 VDC, 24VDC, and unregulated battery voltage (48 - 60 VDC) for powering Warthog's payloads.
+See the figures below for connector locations and pinouts.
+Ensure you select a contact that's appropriate for the gauge of wire used.
+The mating connector for "USER POWER" is [USBFirewire RR-153274-23050](https://www.usbfirewire.com/parts/rr-153274-23050.html#RR-153274-23050) and the mating connector for "SERIAL" is [USBFirewire RR-153274-23040](https://www.usbfirewire.com/parts/rr-153274-23040.html#RR-153274-23040).
+
+<center>
+  <img
+    src="/img/robot_images/warthog_images/robot_warthog_2.png"
+    width="600"
+  />
+</center>
+
+<center>
+  <img
+    src="/img/robot_images/warthog_images/robot_warthog_3.png"
+    width="600"
+  />
+</center>
+
+The electrical system for the chassis can be seen below.
+
+<center>
+  <img
+    src="/img/robot_images/warthog_images/robot_warthog_4.png"
+    width="400"
+  />
+</center>
+
+The left and right drive units electrical system is decribed in the following figures.
+
+<center>
+  <img
+    src="/img/robot_images/warthog_images/robot_warthog_5.png"
+    width="400"
+  />
+</center>
+
+<center>
+  <img
+    src="/img/robot_images/warthog_images/robot_warthog_6.png"
+    width="400"
+  />
+</center>
+
+### Software Integration
+
+ROS has a large ecosystem of sensor drivers, some of which include pre-made URDF description and even
+simulation configurations. Please see the following page on the ROS wiki for a partial list:
+
+<http://wiki.ros.org/Sensors>
+
+For the best experience, consider purchasing supported accessories from Clearpath Robotics for your Warthog.
+This will include simulation, visualization, and driver support for those accessories.
+However, we will happily help you get started with integrating your own devices as well.
+
+---
+
+## Maintenance
```

Comment:
Are we going to keep that in the User Manual or break this out into the separate document that you had proposed?

Reply:
I will move the Maintenance content to the separate page.